src/script/view_model/ListViewModel.ts

```diff
@@ -123,7 +123,7 @@ export class ListViewModel {
       const hasConnectRequests = !!this.userState.connectRequests().length;
       const states: (string | Conversation)[] = hasConnectRequests ? [ContentState.CONNECTION_REQUESTS] : [];
 
-      return states.concat(this.conversationState.unarchivedConversations());
+      return states.concat(this.conversationState.visibleConversations());
```

Comment:
In theory it's a little less performant since you need to duplicate (by spreading) the initial array and then create a new one. 
`.concat` will only create a single new array